bazel/bazel_test.go

```diff
@@ -94,26 +94,31 @@ func TestCancel(t *testing.T) {
 }
 
 var bazelNpmPathTests = []struct {
-	in string
+	in  string
 	out string
 	err error
 }{
 	{"/node_modules/@bazel/ibazel/bin/linux_amd64/ibazel", os.Getenv("TEST_TMPDIR") + "/node_modules/@bazel/bazel-linux_x64/bazel-1.2.3-linux_x86_64", nil},
 	{"/node_modules/@bazel/ibazel/bin/windows_amd64/ibazel.exe", os.Getenv("TEST_TMPDIR") + "/node_modules/@bazel/bazel-windows_x64/bazel-1.2.3-windows_x86_64.exe", nil},
 	{"/node_modules/@bazel/ibazel/bin/darwin_amd64/ibazel", os.Getenv("TEST_TMPDIR") + "/node_modules/@bazel/bazel-darwin_x64/bazel-1.2.3-darwin_x86_64", nil},
+	{"/node_modules/@bazel/ibazel/bin/darwin_arm64/ibazel", os.Getenv("TEST_TMPDIR") + "/node_modules/@bazel/bazel-darwin_arm64/bazel-1.2.3-darwin_arm64", nil},
 	{"/", "", errors.New("bazel binary not found in @bazel/bazel package")},
 }
+
 func TestBazelNpmPath(t *testing.T) {
 	// Where bazel gets installed by npm
 	bazelNpmDir := os.Getenv("TEST_TMPDIR") + "/node_modules/@bazel"
-	
-	if err := os.MkdirAll(bazelNpmDir + "/bazel-linux_x64", 0755); err != nil {
+
+	if err := os.MkdirAll(bazelNpmDir+"/bazel-linux_x64", 0755); err != nil {
+		t.Errorf(err.Error())
+	}
+	if err := os.MkdirAll(bazelNpmDir+"/bazel-windows_x64", 0755); err != nil {
 		t.Errorf(err.Error())
 	}
-	if err := os.MkdirAll(bazelNpmDir + "/bazel-windows_x64", 0755); err != nil {
+	if err := os.MkdirAll(bazelNpmDir+"/bazel-darwin_x64", 0755); err != nil {
 		t.Errorf(err.Error())
 	}
-	if err := os.MkdirAll(bazelNpmDir + "/bazel-darwin_x64", 0755); err != nil {
+	if err := os.MkdirAll(bazelNpmDir+"/bazel-darwin_arm64", 0755); err != nil {
```

Comment:
Please run `gofmt` on this file

Reply:
I've run `gofmt -s -w ./bazel/bazel_test.go` but it does not change anything on the file